src/DataModel/ProductData.js

```diff
@@ -0,0 +1,17 @@
+const products = [
+  {
+    name: 'Iphone 11 Pro (Silver)', price: 999, storage: '64gb', image: 'silver',
```

Comment:
@prabhatVinsol add random uuid here which can be used for key

Reply:
Done